frontend/public/components/storage-class-form.tsx

```diff
@@ -542,6 +549,11 @@ export class StorageClassForm_ extends React.Component<StorageClassFormProps, St
       data.reclaimPolicy = this.state.newStorageClass.reclaim;
     }
 
+    const volumeBindingMode = _.get(this.storageTypes[type], 'volumeBindingMode', null);
+    if (volumeBindingMode !== null) {
```

Comment:
Unless there's a reason we would ever need to add values like empty string to the request, prefer

```suggestion
    if (volumeBindingMode) {
```